pkgs/stdenv/generic/setup.sh

```diff
@@ -422,50 +421,55 @@ fi
 
 
 substitute() {
-    local input="$1"
-    local output="$2"
+    local input=$1
+    local output=$2
```

Comment:
I basically got bash Stockholm syndrome writing this stuff. Happy to quote everything for consistency.